### PR TITLE
Add eCF directory

### PR DESCRIPTION
**Pull request by KChen1250**
Add MathematicaToLaTeX.py which converts Identities.m to a LaTeX file (newIdentities.m) using the macros stated in Glossary.csv

### CODE REVIEW

**Comment by notjagan** [comment]
write all the tests

**Comment by joonbang** [comment]
You should go back and delete Identities.m from your git history, not just from your repo.

**Comment by KChen1250** [comment]
@joonbang I think you're talking about IdentitiesTest.m, which is irrelevant to Identities.m

**Comment by HowardCohl** [comment]
@KChen1250 @joonbang ```Identities.m``` is private data and definitely should not be made public.

Are we talking about the same thing?

**Comment by KChen1250** [comment]
@HowardCohl @joonbang I talked to Joon; he mistook `IdentitiesTest.m` for `Identities.m`, which is just my test file and isn't private data;` Identities.m` is still safe at NIST and is not public.

**Comment by KChen1250** [comment]
Yay it's above 50% now

**Comment by notjagan** [comment]
hype

**Comment by KChen1250** [comment]
over 99% yay

File: eCF/.gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# ecf project
+*.sty
+Identities.*
+newIdentities.*
+*.csv
+macros.txt
+*.log
+*.aux
+ZE.3.*
```
Comment (notjagan) [inline eCF/.gitignore]: Newline at the end of the file.

File: eCF/test/test_find_surrounding.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+__author__ = 'Kevin Chen'
+__status__ = 'Development'
+
+from unittest import TestCase
+from MathematicaToLaTeX import find_surrounding
+
```
Comment (notjagan) [inline eCF/test/test_find_surrounding.py]: What does this file do... ?

**Pull request merged**